samples/msal-node-samples/refresh-token/adalApp.js

```diff
@@ -0,0 +1,99 @@
+// Import dependencies
+var express = require('express');
```

Comment:
nitpick: Do we have a style guide? I'm wondering why you used single quotes instead of double quotes.

Reply:
Node.js seems to favour single quotes in their [own documentation](https://nodejs.org/en/docs/guides/getting-started-guide/), although I don't think they have a public style guide